homeassistant/components/alarm_control_panel/manual_mqtt.py

```diff
@@ -5,25 +5,21 @@
 https://home-assistant.io/components/alarm_control_panel.manual_mqtt/
 """
 import asyncio
-import datetime
 import logging
 
 import voluptuous as vol
 
-import homeassistant.components.alarm_control_panel as alarm
-import homeassistant.util.dt as dt_util
+import homeassistant.components.alarm_control_panel.manual as manual_alarm
```

Comment:
I don't think it's allowed for a platform to import and depend on another platform.